src/test/java/lambda/part1/exercise/Exercise3.java

```diff
@@ -0,0 +1,72 @@
+package lambda.part1.exercise;
+
+import com.google.common.collect.FluentIterable;
+import lambda.data.Person;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+@SuppressWarnings({"ConstantConditions", "unused"})
+public class Exercise3 {
+
+    @Test
+    public void sortPersonsByAgeUsingArraysSortExpressionLambda() {
+        Person[] persons = getPersons();
+
+        // TODO использовать Arrays.sort + expression-lambda
+        Arrays.sort(persons, (firstPerson, secondPerson) ->
+                firstPerson.getAge() - secondPerson.getAge());
```

Comment:
It's better to use ```Integer.compare(..., ...)``` because of potential overflow. 
See: https://stackoverflow.com/a/2728810